corehq/apps/app_manager/static/app_manager/js/app_view.js

```diff
@@ -3,6 +3,15 @@ hqDefine("app_manager/js/app_view", function () {
     $(function () {
         var initialPageData = hqImport("hqwebapp/js/initial_page_data");
 
+        // App name
+        $(document).on("inline-edit-save", function (e, data) {
+            if (_.has(data.update, '.variable-app_name')) {
+                var appManager = hqImport('app_manager/js/app_manager');
+                appManager.updatePageTitle(data.update['.variable-app_name']);
```

Comment:
One thing that is not clear in this PR (this PR did not make it worse) is where `.variable-...` values originate. In other words, it is not clear what code creates those "variables"? On the other hand, they are easy to find with a tool like grep.

Reply:
I'm going to leave this alone, I think the grep is sufficient.